inform7/extensions/basic_inform/Sections/Glulx and Glk.w

```diff
@@ -0,0 +1,225 @@
+Glulx and Glk.
+
+Support for Glulx and Glk interpreter systems.
+
+@ =
+Part Four - Glulx and Glk (for Glulx only)
+
+@h Feature testing.
+These phrases let us test for various interpreter features.
+While most features can use the generic functions, a few need special handling,
+and so individual phrases are defined for them.
```

Comment:
There is a unicode character kind so it would be possible to add phrases for `gestalt_CharOutput`, `gestalt_CharInput`, and `gestalt_LineInput`. But there would be two complications: first, the function keys aren't valid unicode characters. Second, many interpreters return results based on whether they'll crash or not. In practice they say everything can be printed and read. But most authors would probably expect it to instead be about whether the font supports the character, which most interpreters are not checking. So I'm inclined to just leave these gestalts out.

Reply:
I agree; those gestalt selectors have never been good for much.